src/packages/excalidraw/index.tsx

```diff
@@ -33,6 +33,7 @@ const Excalidraw = (props: ExcalidrawProps) => {
     detectScroll = true,
     handleKeyboardGlobally = false,
     onLibraryChange,
+    autofocus,
```

Comment:
```suggestion
    autofocus = false,
```
lets make it `false` by default and required prop for `App.tsx`